package/src/i18n/es.json

```diff
@@ -0,0 +1,70 @@
+  "Emoji matching": "Coincidencia de emoji",
+  "Empty message...": "Mensaje vacío...",
+  "Error loading": "Error al cargar",
+  "Error loading channel list...": "Error al cargar la lista de canales...",
+  "Error loading messages for this channel...": "Error al cargar los mensajes de este canal...",
+  "Error while loading, please reload/refresh": "Error al cargar, por favor recarga/actualiza",
+  "File type not supported": "Tipo de archivo no admitido",
+  "Flag": "Reportar",
+  "Flag Message": "Reportar mensaje",
+  "Flag action failed either due to a network issue or the message is already flagged": "El reporte falló debido a un problema de red o el mensaje ya fue reportado.",
+  "Instant Commands": "Comandos instantáneos",
+  "Links are disabled": "Los enlaces están desactivados",
+  "Loading channels...": "Cargando canales...",
+  "Loading messages...": "Cargando mensajes...",
+  "Loading...": "Cargando...",
+  "Message Reactions": "Reacciones al mensaje",
+  "Message deleted": "Mensaje eliminado",
+  "Message flagged": "Mensaje reportado",
+  "Mute User": "Silenciar usuario",
+  "Not supported": "No admitido",
+  "Nothing yet...": "Aún no hay nada...",
+  "Ok": "Aceptar",
+  "Only visible to you": "Solo visible para ti",
+  "Photo": "Foto",
+  "Photos": "Fotos",
+  "Photos and Videos": "Fotos y videos",
+  "Pin to Conversation": "Fijar a la conversación",
+  "Pinned by": "Fijado por",
+  "Please enable access to your photos and videos so you can share them.": "Por favor, habilita el acceso a tus fotos y videos para poder compartirlos.",
+  "Please select a channel first": "Por favor, selecciona primero un canal",
+  "Reconnecting...": "Reconectando...",
+  "Reply": "Responder",
+  "Reply to Message": "Responder al mensaje",
+  "Resend": "Reenviar",
+  "Search GIFs": "Buscar GIFs",
+  "Send a message": "Enviar un mensaje",
+  "Sending links is not allowed in this conversation": "No está permitido enviar enlaces en esta conversación",
+  "Slow mode ON": "Modo lento ACTIVADO",
+  "The message has been reported to a moderator.": "El mensaje ha sido reportado a un moderador.",
+  "Thread Reply": "Respuesta de hilo",
+  "Unblock User": "Desbloquear usuario",
+  "Unknown User": "Usuario desconocido",
+  "Unmute User": "Activar sonido del usuario",
+  "Unpin from Conversation": "Desmarcar de la conversación",
+  "Unread Messages": "Mensajes no leídos",
+  "Video": "Video",
+  "You": "Tú",
+  "You can't send messages in this channel": "No puedes enviar mensajes en este canal",
+  "{{ firstUser }} and {{ nonSelfUserLength }} more are typing": "{{ firstUser }} y {{ nonSelfUserLength }} más están escribiendo",
+  "{{ index }} of {{ photoLength }}": "{{ index }} de {{ photoLength }}",
+  "{{ replyCount }} Replies": "{{ replyCount }} Respuestas",
+  "{{ replyCount }} Thread Replies": "{{ replyCount }} respuestas de hilo",
+  "{{ user }} is typing": "{{ user }} está escribiendo",
+  "🏙 Attachment...": "🏙 Adjunto..."
+}
```

Comment:
We just added a new translations for `Maximum file size upload limit reached. Please upload a file below {{MAX_FILE_SIZE_TO_UPLOAD_IN_MB}} MB.` and there is one coming for `Select More Photos` in https://github.com/GetStream/stream-chat-react-native/pull/2291/files.

Can you please add them? Thanks 😄

Reply:
No problem. Done 👍